### PR TITLE
Various fixes & additions for library assets

### PR DESCRIPTION
**Pull request by thisandagain**
### Resolves
Resolves GH-2310
Resolves GH-2349
Resolves GH-2375

### Proposed Changes
- Changes name of the "theatre" backdrop to "theater"
- Resolves issue with "heart" backdrops
- Adds tags to ice skates (note: this will likely be removed / replaced soon
- Adds "baseball" theme
- Adds "spooky" theme (replaces old ghosts, bats, and woods backdrop)

### Browser Coverage
Check the OS/browser combinations tested (At least 2)

Mac
 * [x] Chrome 
 * [x] Firefox 
 * [ ] Safari
 
Windows
 * [ ] Chrome 
 * [ ] Firefox 
 * [ ] Edge
 
Chromebook
 * [ ] Chrome
 
iPad
* [ ] Safari

Android Tablet
* [ ] Chrome

### CODE REVIEW

File: src/lib/libraries/backdrops.json [hunk 1]
```diff
@@ -215,7 +247,7 @@
     },
     {
         "name": "Jurassic ",
-        "md5": "73fd0531a1aaca5dff2d1c67202ff5bd.svg",
+        "md5": "2d59bed41204445ae9dbcb963b8da14f.svg",
```
Comment (fsih) [inline src/lib/libraries/backdrops.json]: Did you mean to change jurassic? It seems to be off-center now

File: src/lib/libraries/backdrops.json [hunk 2]
```diff
@@ -563,5 +595,24 @@
...
         ]
+    },
+    {
+        "name": "Woods",
+        "md5": "3f3005b76e278b66dbeb9a982d68c555.svg",
+        "type": "backdrop",
+        "tags": [
+            "fantasy",
+            "spooky",
+            "holloween",
```
Comment (fsih) [inline src/lib/libraries/backdrops.json]: "halloween" (also other occurrences)

File: src/lib/libraries/costumes.json [hunk 3]
```diff
@@ -6257,33 +6613,39 @@
     },
     {
         "name": "Shark-a",
-        "md5": "4ca6776e9c021e8b21c3346793c9361d.svg",
+        "md5": "7c0a907eae79462f69f8e2af8e7df828.svg",
```
Comment (fsih) [inline src/lib/libraries/costumes.json]: Did you mean to replace Ipsy's shark with the old sharks?

File: src/lib/libraries/sprites.json [hunk 4]
```diff
@@ -1,7 +1,6 @@
 [
     {
         "name": "Abby",
-        "baseLayerID": -1,
```
Comment (fsih) [inline src/lib/libraries/sprites.json]: Why did all base layer IDs need to be changed to -1? What is baseLayerID used for?

Comment (thisandagain) [reply src/lib/libraries/sprites.json]: This is a change to `libgen` that attempts to normalize all of the assets before the libraries are generated.

File: src/lib/libraries/backdrops.json (hunk 1 above)
Comment (thisandagain) [reply src/lib/libraries/backdrops.json]: Hmm. I didn't change this. I'll check with @kosiecki

File: src/lib/libraries/costumes.json (hunk 3 above)
Comment (fsih) [reply src/lib/libraries/costumes.json]: (Also shark c still exists in the costume library)

**Pull request merged**